connectproto/proposervm/service.proto

```diff
@@ -20,3 +23,13 @@ message GetProposedHeightReply {
   // right now.
   uint64 height = 1;
 }
+
+// Request to get the current epoch.
+message GetCurrentEpochRequest {}
+
+// Response containing the current epoch.
+message GetCurrentEpochReply {
+  uint64 p_chain_height = 1;
+  uint64 number = 2;
```

Comment:
Nit: I don't think it matters either way for the proto definition here, but it seems a bit odd to have the `p_chain_height` index as the first field and `number` as the second field when `number` is the unique identifier and `p_chain_height` is not necessarily unique.

Reply:
+1